### ISSUE TITLE
Mouse graphic is showing a wired mouse despite using a wireless mouse

### ISSUE THREAD

**Issue by RenateUSB** [LibreCAD/LibreCAD#1956]
Seriously, that mouse graphic is horrible and takes up more space than is necessary.
As an element that can't be disabled it eats a slice off the entire width of your monitor.
Can't you have a simpler graphic rendition?

**Comment by sand1024** [comment]
hm... just trying to clarify. 
By "mouse graphic" - you men mouse icon in the status bar, right (between left/right button prompts)? 
If it so, could you please also share a screenshot with status bar? 

And couple of additional questions: 
1) statusbar - is it in fixed "classic" or toolbar-like mode? 
2) If statusbar is in classic mode, are there custom settings for status bar height/fonts in Widget options? 

Basically, it's possible to improve that section for status bar in "non-classic" mode. Say, disable prompts entire (together with mouse graphic) if prompts are shown in bottom status bar. 

In my setup this icons shown nicely, so I need to understand how it behaves in your environment. 

![image](https://github.com/user-attachments/assets/8ac0f377-2434-45de-9a68-87cc1ef44ef1)
